models/issue.go

```diff
@@ -63,6 +63,7 @@ type Issue struct {
 	Reactions        ReactionList  `xorm:"-"`
 	TotalTrackedTime int64         `xorm:"-"`
 	Assignees        []*User       `xorm:"-"`
+	ProjectIssueID   ProjectIssue
```

Comment:
```suggestion
	ProjectIssueID   ProjectIssue `xorm:"-"`
```

Reply:
How is this resolved?

Why is ProjectIssueID type ProjectIssue - this is not good. It should be the type of ProjectIssue.ID field.

Reply:
@patcito still not applyed